src/Products/Annotation/Annotator/DistanceAnnotator.cs

```diff
@@ -1,81 +1,95 @@
-﻿using GroupDocs.Annotation.Domain;
-using GroupDocs.Annotation.Domain.Containers;
+﻿using GroupDocs.Annotation.Models;
+using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
+using System.Globalization;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
-    public class DistanceAnnotator : AbstractBoxAnnotator
+    public class DistanceAnnotator : BaseAnnotator
     {
+        private DistanceAnnotation distanceAnnotation;
 
-        public DistanceAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
+        public DistanceAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
+            distanceAnnotation = new DistanceAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)